src/main/java/appeng/client/gui/implementations/GuiInterfaceTerminal.java

```diff
@@ -39,21 +45,29 @@
 import net.minecraft.nbt.NBTTagList;
 
 import appeng.api.AEApi;
+import appeng.api.config.SearchBoxMode;
+import appeng.api.config.Settings;
+import appeng.api.config.TerminalStyle;
 import appeng.client.gui.AEBaseGui;
+import appeng.client.gui.widgets.GuiImgButton;
 import appeng.client.gui.widgets.GuiScrollbar;
 import appeng.client.gui.widgets.MEGuiTextField;
 import appeng.client.me.ClientDCInternalInv;
 import appeng.client.me.SlotDisconnected;
 import appeng.container.implementations.ContainerInterfaceTerminal;
+import appeng.core.AEConfig;
+import appeng.core.AELog;
 import appeng.core.localization.GuiText;
+import appeng.core.sync.network.NetworkHandler;
+import appeng.core.sync.packets.PacketValueConfig;
+import appeng.integration.Integrations;
 import appeng.parts.reporting.PartInterfaceTerminal;
 import appeng.util.Platform;
 
 
 public class GuiInterfaceTerminal extends AEBaseGui
 {
-
-	private static final int LINES_ON_PAGE = 6;
+	private final static int SECTION_HEIGHT = 18;
 
 	// TODO: copied from GuiMEMonitorable. It looks not changed, maybe unneeded?
 	private final int offsetX = 9;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unread field: appeng.client.gui.implementations.GuiInterfaceTerminal.offsetX; should this field be static? [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=findbugs%3ASS_SHOULD_BE_STATIC)